app/mailers/alaveteli_pro/metrics_mailer.rb

```diff
@@ -0,0 +1,35 @@
+module AlaveteliPro
+  ##
+  # A mailer responsible for sending out the weekly metrics report to
+  # the Pro contact
+  #
+  class MetricsMailer < ApplicationMailer
+    require_relative '../../../lib/alaveteli_pro/metrics_report.rb'
```

Comment:
I loaded it separately as it's not used in the app itself, only by a mailer script, so it didn't feel useful to have it in memory except for once a week, particularly as it's accessing potentially sensitive customer account info.

But it's easy enough to move it if you think it would be more logical to load it from `config/initializers/alaveteli.rb` instead.